### PR TITLE
AC-963 : In Patient Registration, date of birth edit text's keyboard/input type does not have forward slash

### PR DESCRIPTION
**Pull request by rdodiya**
## Description of what I changed

In Patient Registration, make Calender ImageButton visible by adding backgroundTint to ImageButton of calender. 
## Issue I worked on

Issue : [https://issues.openmrs.org/browse/AC-963](https://issues.openmrs.org/browse/AC-963)

## Test app after changes:

Image 1 :  

Screenshot 2021-09-18 14 57 19

Image 2 : 

Screenshot 2021-09-18 14 57 32

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: .idea/runConfigurations.xml [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <component name="RunConfigurationProducerService">
     <option name="ignoredProducers">
       <set>
+        <option value="com.android.tools.idea.compose.preview.runconfiguration.ComposePreviewRunConfigurationProducer" />
         <option value="org.jetbrains.plugins.gradle.execution.test.runner.AllInPackageGradleConfigurationProducer" />
```
Comment (LuGO0) [inline .idea/runConfigurations.xml:7]: ignore this file

Comment (rdodiya) [reply .idea/runConfigurations.xml:7]: ok should I have to delete it?

**Comment by rdodiya** [comment]
@LuGO0  if there is anything to change then please let me know . If there is nothing to change then please merge merge this  so I can send another PR for another issue

File: .idea/runConfigurations.xml (hunk 1 above)
Comment (LuGO0) [reply .idea/runConfigurations.xml:7]: Add it to .gitignore

Comment (rdodiya) [reply .idea/runConfigurations.xml:7]: ok  it's done . Sorry I didn't  get it first.

**Comment by LuGO0** [comment]
The file is still being tracked @rdodiya, do one thing close this PR and try to follow this article step by step while making a new PR https://wiki.openmrs.org/display/docs/Pull+Request+Tips . and dont add any files in commit which are not required.

**Pull request closed without merging**